Engine/Component/ComponentCamera.cpp

```diff
@@ -208,6 +209,7 @@ void ComponentCamera::RecordFrame(float width, float height)
 	}
 
 	App->renderer->RenderFrame(*this);
+	App->artificial_intelligence->RenderNavMesh(*App->cameras->scene_camera);
```

Comment:
This should be drawm in `ModuleDebugDraw` in order to keep responsabilities isolated in classes.